src/app/components/schedule-card/ScheduleCard.scss

```diff
@@ -98,13 +105,23 @@
 				}
 			}
 
-			h5 {
-				font-size: 21px;
+			.name {
```

Comment:
since we're using straight scss files and not css modules these class rules will apply to literally anything and everything with the class name `name` in the vh website; we should pick something more specific (like adding `schedule-card-` as a prefix to these classnames or something) so it's easier to troubleshoot if our css rules are working as intended everywhere (same with `.time` here and `.container-style` for the container surrounding the countdown)